src/js/components/help/helpContent.jsx

```diff
@@ -66,8 +66,17 @@ export default class HelpContent extends React.Component {
         if(this.props.helpOnly){
             membership = <div>
             <h4 name='agencyAccess'>Request Agency Access</h4>
-
-            <p>Click here to <a href='https://community.max.gov/x/fJwuRQ' target="_blank">request agency access</a></p>
+            <p>
+                The DATA Act Broker leverages MAX.gov for user accounts and password authentication. 
+                To request a Broker account, you will first need a MAX.gov account. 
+                If you do not have a MAX.gov account, you may register for one <a href='https://max.gov/maxportal/registrationForm.action' target='_blank'>here</a>.
+            </p>
```

Comment:
The line breaks were for readability purposes. It is broken up to follow the formatting